src/Livewire/FolderTree.php

```diff
@@ -44,16 +46,41 @@ public function loadModel(array $arguments): void
         $this->js(<<<'JS'
             selected = false;
             loadLevels();
-        JS);
+        JS
```

Comment:
> 
        $this->js(
            <<<'JS'
               selected = false;
               loadLevels();
            JS
        );